components/entitlement/org.wso2.carbon.identity.api.server.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/endpoint/auth/BasicAuthHandler.java

```diff
@@ -114,8 +114,10 @@ public boolean isAuthenticated(ContainerRequestContext message) {
                             authzHeaders.set(0, userName);
                             return true;
                         } else {
-                            log.error("Authentication failed for the user: " + tenantLessUserName
-                                    + "@" + tenantDomain);
+                            if (log.isDebugEnabled()) {
+                                log.debug("Authentication failed for the user: " + tenantLessUserName
```

Comment:
let's use the term 'tenantAwareUsername' instead, which is used in other places

Reply:
Fixed